main/acle.md

```diff
@@ -10922,114 +11001,214 @@ BFloat16 floating-point multiply-subtract long from single-precision (top/bottom
     __arm_streaming_compatible;
   ```
 
-#### SMAX, SMIN, UMAX, UMIN, FMAX, FMIN (single)
+#### SMAX, SMIN, UMAX, UMIN, BFMAX, BFMIN, FMAX, FMIN (single)
 
 Multi-vector min/max
 
 ``` c
-  // Variants are also available for _single_s8_x2, _single_u8_x2,
-  // _single_s16_x2, _single_u16_x2, _single_s32_x2, _single_u32_x2,
-  // _single_f32_x2, _single_s64_x2, _single_u64_x2 and _single_f64_x2
+  // Variants are also available for:
+  //   _single_s8_x2
+  //   _single_u8_x2,
+  //   _single_bf16_x2 (only if __ARM_FEATURE_SME_B16B16)
+  //   _single_s16_x2
+  //   _single_u16_x2
```

Comment:
I believe the SME2 ACLE omitted sv[max|min|maxnm|minnm]_single_f16_x2 and  sv[max|min|maxnm|minnm]_f16_x2

Reply:
The f16 versions are the exemplars:
```
  svfloat16x2_t svmax[_single_f16_x2](svfloat16x2_t zdn, svfloat16_t zm)
    __arm_streaming;
```